### ISSUE TITLE
[1.10.2] Error with ThermalFoundation

### ISSUE THREAD

**Issue by essemX** [dan200/ComputerCraft#388]
Originally reported there:
https://github.com/CoFH/Feedback/issues/419

Stacktrace:
```
java.lang.IllegalArgumentException: Cannot get property PropertyEnum{name=type, clazz=class cofh.thermalfoundation.block.BlockOreFluid$Type, values=[CRUDE_OIL_SAND, CRUDE_OIL_GRAVEL, REDSTONE, GLOWSTONE, ENDER]} as it does not exist in BlockStateContainer{block=minecraft:stone, properties=[variant]}
        at net.minecraft.block.state.BlockStateContainer$StateImplementation.func_177229_b(BlockStateContainer.java:196)
        at cofh.thermalfoundation.block.BlockOreFluid.func_176201_c(BlockOreFluid.java:86)
        at cofh.thermalfoundation.block.BlockOreFluid.func_149744_f(BlockOreFluid.java:134)
        at dan200.computercraft.shared.util.RedstoneUtil.getRedstoneOutput(RedstoneUtil.java:60)
        at dan200.computercraft.shared.computer.blocks.TileComputerBase.updateInput(TileComputerBase.java:329)
        at dan200.computercraft.shared.turtle.core.TurtleBrain.teleportTo(TurtleBrain.java:528)
        at dan200.computercraft.shared.turtle.core.TurtleMoveCommand.execute(TurtleMoveCommand.java:112)
        at dan200.computercraft.shared.turtle.core.TurtleBrain.updateCommands(TurtleBrain.java:998)
        at dan200.computercraft.shared.turtle.core.TurtleBrain.update(TurtleBrain.java:185)
        at dan200.computercraft.shared.turtle.blocks.TileTurtle.func_73660_a(TileTurtle.java:259)
        at org.spongepowered.common.event.tracking.TrackingUtil.tickTileEntity(TrackingUtil.java:231)
        at net.minecraft.world.WorldServer.updateTileEntity(WorldServer.java:2831)
        at net.minecraft.world.WorldServer.redirect$onUpdateTileEntities$zka000(WorldServer.java:2818)
        at net.minecraft.world.World.func_72939_s(World.java:6049)
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:2185)
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:709)
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:387)
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613)
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471)
        at java.lang.Thread.run(Thread.java:748)```

Versions:
ComputerCraft1.80pr0
CoFHCore-1.10.2-4.1.11.16-universal
ThermalFoundation-1.10.2-2.1.5.12-universal

**Comment by SquidDev** [comment]
I'm pretty sure this was fixed in c7f5d039b2c8f478cc8e8250b26a1e762dc53f5d, but it hasn't been officially released yet. Were you using the download on Curse? If so, [try this version instead](https://cc.crzd.me/maven/dan200/computercraft/ComputerCraft/1.80pr0-build12/ComputerCraft-1.80pr0-build12.jar).

**Comment by dan200** [comment]
Gonna take SquidDev's word for it this is fixed
